public/components/notebooks/components/notebook.tsx

```diff
@@ -537,9 +543,9 @@ export class Notebook extends Component<NotebookProps, NotebookState> {
     }
   };
 
-  runForAllParagraphs = (reducer: (para: ParaType, index: number) => Promise<any>) => {
+  runForAllParagraphs = (reducer: (para: ParaType, _index: number) => Promise<any>) => {
     return this.state.parsedPara
-      .map((para: ParaType, index: number) => () => reducer(para, index))
+      .map((para: ParaType, _index: number) => () => reducer(para, _index))
```

Comment:
Is there a reason we update _index to indicate it's unused, while using it in reducer?

Reply:
The linter is not able to understand that index is used in reducer and leads to `no-unused-vars` error. So marked this one as an internal variable.